config/config.go

```diff
@@ -31,16 +31,17 @@ func New() Config {
 }
 
 type Config struct {
+	AccessLog  Log `envconfig:"ACCESS_LOG"`
+	ErrorLog   Log `envconfig:"ERROR_LOG"`
+	Debug      bool
+	Origin     string
 	Port       string `default:"8080"`
-	SSE        ServerSideEvent
+	SSE        ServerSentEvent
 	Subscriber Subscriber
-	ErrorLog   Log  `envconfig:"ERROR_LOG"`
-	AccessLog  Log  `envconfig:"ACCESS_LOG"`
 	TLS        Cert `envconfig:"TLS"`
-	Origin     string
 }
 
-type ServerSideEvent struct {
+type ServerSentEvent struct {
```

Comment:
__[golint]__
* [golint] exported type ServerSentEvent should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->